tests/conftest.py

```diff
@@ -0,0 +1,15 @@
+import os
```

Comment:
Would have preferred to see the test-setup part in a separate PR, but I guess it was blocking for adding the tests ;)

Reply:
I can pull it out. :)

Reply:
Nah, it looks good to me :)

Reply:
https://github.com/tmrowco/brick/pull/65 :)